web/src/lib/components/admin-page/settings/setting-switch.svelte

```diff
@@ -1,12 +1,19 @@
 <script lang="ts">
   import { quintOut } from 'svelte/easing';
   import { fly } from 'svelte/transition';
+  import { createEventDispatcher } from 'svelte';
 
   export let title: string;
   export let subtitle = '';
   export let checked = false;
   export let disabled = false;
   export let isEdited = false;
+
+  const dispatch = createEventDispatcher();
+
+  function onToggle(event: Event) {
+    dispatch('toggle', (event.target as HTMLInputElement).checked);
+  }
```

Comment:
I'd be fine inlining this. If you keep it here, we use the form `const onToggle = (event: Event) => { ... }` more often than this function syntax.